packages/modal/stories/Modal.stories.tsx

```diff
@@ -26,6 +26,7 @@ import { TextInput } from "../../textInput";
 import { fullscreenModalTitle } from "../../fullscreenView/style";
 import { blue } from "../../design-tokens/build/js/designTokens";
 import { SystemIcons } from "../../icons/dist/system-icons-enum";
+import { returnStatement } from "@babel/types";
```

Comment:
Looks like you have an accidental unused import here for `returnStatement` that you can remove.

Reply:
Sure